netsim/cli/connect.py

```diff
@@ -167,8 +167,9 @@ def connect_to_node(
       args: argparse.Namespace,
       rest: list,
       topology: Box,
+      node: str,
       log_level: LogLevel = LogLevel.INFO) -> typing.Union[bool,int,str]:
-  node = args.host
+  
```

Comment:
... and then this would be `node = node or args.host`

Anyway, separating **node** from the other arguments makes sense, but then we should put it somewhere closer to the top of the list, maybe even as the first argument (which would require fixing all calls to `connect_to_node`)